en/docs/security-announcements/security-advisories/2023/WSO2-2022-2177.md

```diff
@@ -0,0 +1,162 @@
+| WSO2 Open Banking AM    |           1.3.0 |           16 | 1666409836919 |
+| WSO2 Open Banking AM      |           1.4.0 |           25 | 1666409836919 |
+| WSO2 Open Banking AM   |           1.5.0 |           23 | 1666409836919 |
+| WSO2 Open Banking AM       |           2.0.0 |           202 | 1666621544419 |
+| WSO2 Open Banking AM Accelerator        |           3.0.0 |           Refer to WSO2 AM-4.0.0 update level since this vulnerability is originating from the base product[^3].| N/A |
+| WSO2 Open Banking IS-KM        |           1.3.0 |           88 | 1666409836919 |
+| WSO2 Open Banking IS-KM         |           1.4.0 |           92 | 1666409836919 |
+| WSO2 Open Banking IS-KM              |           1.5.0 |           91 | 1666409836919 |
+| WSO2 Open Banking IAM                |           2.0.0 |           208| 1666621544419 |
+| WSO2 Open Banking IAM Accelerator               |           3.0.0 |          Refer to WSO2 IAM-5.11.0 update level since this vulnerability is originating from the base product [^3]. | N/A |
+
+
+If you are an open-source user or using a product version that is EOL (End of License) :
+
+You may migrate to the latest version of the product if the latest version is not listed under the list of the affected products. Otherwise, you may apply the relevant fixes to the product based on the public fixes as given below:
+
+* [https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216](https://github.com/wso2-extensions/identity-carbon-auth-rest/pull/216)
+
+Or else you may follow the mitigation steps given below.
+
+#### Group 1:
+For below listed products, it is reoccomand to append the following configuration lines to the "&lt;PRODUCT-HOME&gt;/repository/conf/deployment.toml" file and restart the server.
+
+
+- WSO2 Identity Server 5.9.0, 5.10.0, 5.11.0, 6.0.0
+- WSO2 Identity Server as Key Manager 5.9.0, 5.10.0
+- WSO2 API Manager 3.0.0, 3.1.0, 3.2.0, 4.0.0, 4.1.0
+- WSO2 Open Banking AM 2.0.0, 3.0.0
+- WSO2 Open Banking IAM 2.0.0, 3.0.0
+
+
+```toml
+[[resource.access_control]]
+context="(.*)/((\\.+)|(.*;+.*)|%2e)/(.*)"
+permissions = ["/"]
+secure=true
+http_method="all"
+``` 
+If you have already applied the similar configuration in your "deployment.toml", you can add the above given configuration as a new config & restart the server.
+
+#### Group 2:
+For group 2 products, please Add the below given config as the first entry of the `<ResourceAccessControl>` tag of "`<PRODUCT-HOME>`/repository/conf/identity/identity.xml" file and restart the server.
+
+
+- WSO2 Identity Server 5.3.0, 5.4.0, 5.4.1, 5.5.0, 5.6.0, 5.7.0, 5.8.0
+- WSO2 Identity Server as Key Manager 5.3.0, 5.5.0, 5.6.0, 5.7.0
+- WSO2 Open Banking AM 1.3.0, 1.4.0, 1.5.0
+- WSO2 Open Banking IAM 1.3.0, 1.4.0, 1.5.0
+
+```xml
+<Resource context="(.*)/((\.+)|(.*;+.*)|%2e)/(.*)" secured="true" http-method="all">
+            <Permissions>/</Permissions>
+</Resource>
+```
+
+Example:
```

Comment:
Shall we use the example code segment like this? 

```suggestion
!!! example
     ```xml
      <ResourceAccessControl>
          <Resource context="(.*)/((\.+)|(.*;+.*)|%2e)/(.*)" secured="true" http-method="all">
                 <Permissions>/</Permissions>
          </Resource>
          <Resource context="(.*)" secured="false" http-method="OPTIONS"/>
          <Resource context="/" secured="false" http-method="GET"/>
          ......
      </ResourceAccessControl>
      ```
```